main.go

```diff
@@ -92,6 +94,37 @@ func exportArtifacts(artifacts []gradle.Artifact, deployDir string) ([]string, e
 	return paths, nil
 }
 
+func filterUtilityVariants(variants []string) []string {
```

Comment:
It may be more accurate to call this filterNonUtilityVariants as that is returned :|